src/chrome/content/rules/Connect.me.xml

```diff
@@ -1,22 +1,16 @@
-<!--
-	^connect.me: Expired, missing certificate chain
-	www.connect.me: Refused
-
--->
 <ruleset name="Connect.me">
 
 	<!--	Complications:
 				-->
-  <target host="connect.me"/>
```

Comment:
We use to add comments for non-working hosts. Please document that `connect.me` serves an incomplete certificate chain.